checkov/terraform/checks/resource/aws/SagemakerEndpointConfigurationNameSpecified.py

```diff
@@ -0,0 +1,19 @@
+from checkov.common.models.consts import ANY_VALUE
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+
+class SagemakerEndpointConfigurationEndpointNameSpecified(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure Amazon SageMaker endpoint has a name specified"
+        id = "CKV_AWS_375"
+        supported_resources = ['aws_sagemaker_endpoint_configuration']
+        categories = [CheckCategories.AI_AND_ML]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def get_inspected_key(self):
+        return "name"
```

Comment:
Requiring `name` isn't really a security check. You can also use `name_prefix` which would be valid but flagged by this check.

Reply:
See comment below. 😄